src/current/cockroachcloud/resource-usage.md

```diff
@@ -101,9 +185,63 @@ Initial data ingestion during a migration may consume a high number of RUs. Gene
+
+</section>
+
+## General tips for reducing RU usage
+
+<div class="filters clearfix">
+  <button class="filter-button" data-scope="standard">CockroachDB Standard</button>
+  <button class="filter-button" data-scope="basic">CockroachDB Basic</button>
+</div>
+
+<section class="filter-content" markdown="1" data-scope="standard">
+
+RU consumption can be broken down to more granular components that can give additional optimization insight. Use the {{ site.data.products.cloud }} Console [**Request Units** metrics]({% link cockroachcloud/metrics-request-units.md %}) charts to see what kind of resource usage is driving RU consumption.
 
-### General tips for reducing RU usage
+The following recommendations can help minimize the RU cost of a query by reducing the work your cluster must do to execute that query. The tips are grouped according to the type of resource usage.
+
+### Reads
+
+If the [**Reads** chart]({% link cockroachcloud/metrics-request-units.md %}#reads) shows high RU's for batches, requests, or bytes (payload):
+
+- Use [secondary indexes](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/schema-design-indexes) that reduce the number of rows that need to be scanned.
+- Avoid “wide” columns in tables that are frequently scanned, such as large text or [binary]({% link {{site.current_cloud_version}}/bytes.md %}) columns. Instead, offload them to a separate table that is only accessed when those columns are needed.
+- Don't disable [automatic statistics]({% link {{ site.current_cloud_version }}/cost-based-optimizer.md %}#table-statistics), as they are needed to power the [optimizer](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/cost-based-optimizer).
+- Take advantage of SQL [filters]({% link {{ site.current_cloud_version }}/select-clause.md %}#filter-rows), [joins]({% link {{ site.current_cloud_version }}/joins.md %}), and [aggregations]({% link {{ site.current_cloud_version }}/select-clause.md %}#aggregate-functions) rather than performing these operations in the application to reduce the amount of data returned to the client.
+
+### Writes
+
+If the [**Writes** chart]({% link cockroachcloud/metrics-request-units.md %}#writes) shows high RU's for batches, requests, or bytes (payload):
+
+- Drop indexes that are no longer needed since each additional index requires an additional write.
+- Use [batched `INSERT`](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/insert#bulk-inserts) statements to insert multiple rows in a single statement, rather than sending a separate statement per row.
+- Use range `UPDATE` and `DELETE` statements to affect many rows in a single statement, rather than sending a separate statement per row.
```

Comment:
UPDATE and DELETE can link to those statement docs

Reply:
updated